tests/Feature/Commands/ModelMakeCommandTest.php

```diff
@@ -84,20 +82,20 @@
         $migrationName = now()->format('Y_m_d_his') . '_' . $tableName . '.php';
 
         expect(value: fileExists(relativeFileName: $fileName))->toBeTrue()
-            ->and(File::exists(path: base_path('database/migrations/' . $migrationName)))->toBeTrue();
-
-        expect(
-            value: Str::contains(
-                haystack: fileGet(relativeFileName: $fileName),
-                needles: ['{{ class }}', '{{ namespace }}']
-            )
-        )->toBeFalse()
+            ->and(File::exists(path: base_path('database/migrations/' . $migrationName)))->toBeTrue()
+            ->and(
+                value: Str::contains(
+                    haystack: fileGet(relativeFileName: $fileName),
+                    needles: ['{{ class }}', '{{ namespace }}']
+                )
+            )->toBeFalse()
             ->and(
                 value: Str::contains(
                     haystack: File::get(base_path('database/migrations/' . $migrationName)),
                     needles: ['{{ table_name }}']
                 )
             )->toBeFalse();
+
```

Comment:
and for the Model too

Reply:
model ady covered since they live in src